packages/client/components/DashboardAvatars/DashboardAvatar.tsx

```diff
@@ -37,12 +37,12 @@ const StyledAvatar = styled(Avatar)<{isConnected: boolean; picture: string}>(
 
 const DashboardAvatar = (props: Props) => {
   const {teamMember} = props
-  const {id: teamMemberId, picture, teamId} = teamMember
+  const {id: teamMemberId, picture, teamId, preferredName} = teamMember
   const {user} = teamMember
   if (!user) {
     throw new Error(`User Avatar unavailable. ${JSON.stringify(teamMember)}`)
   }
-  const {isConnected, preferredName} = user
+  const {isConnected} = user
```

Comment:
Did you check why preferredName on User was empty?

Reply:
I'm not well versed with GraphQL, but I noticed `packages/server/graphql/types/User.ts` does have `preferredName` field mentioned. Very curious as to why it did not return. 🤔

Reply:
No worries. I checked and it's because we don't query this field in the notification for new team members
https://github.com/ParabolInc/parabol/blob/6fcd60a5924577cde641b8b19b9a619ab0b12b79/packages/client/mutations/AcceptTeamInvitationMutation.ts#L31